sdk/communication/azure-communication-administration/src/main/java/com/azure/communication/administration/PhoneNumberAsyncClient.java

```diff
@@ -4,48 +4,33 @@
 
 import com.azure.communication.administration.implementation.PhoneNumberAdminClientImpl;
 import com.azure.communication.administration.implementation.PhoneNumberAdministrationsImpl;
-import com.azure.communication.administration.models.AcquiredPhoneNumber;
-import com.azure.communication.administration.models.AreaCodes;
-import com.azure.communication.administration.models.CreateSearchOptions;
-import com.azure.communication.administration.models.CreateSearchResponse;
-import com.azure.communication.administration.models.LocationOptionsQueries;
-import com.azure.communication.administration.models.LocationOptionsQuery;
-import com.azure.communication.administration.models.LocationOptionsResponse;
-import com.azure.communication.administration.models.NumberConfigurationPhoneNumber;
-import com.azure.communication.administration.models.NumberConfigurationResponse;
-import com.azure.communication.administration.models.NumberUpdateCapabilities;
-import com.azure.communication.administration.models.PhoneNumberCountry;
-import com.azure.communication.administration.models.PhoneNumberEntity;
-import com.azure.communication.administration.models.PhoneNumberRelease;
-import com.azure.communication.administration.models.PhonePlan;
-import com.azure.communication.administration.models.PhonePlanGroup;
-import com.azure.communication.administration.models.PstnConfiguration;
-import com.azure.communication.administration.models.ReleaseRequest;
-import com.azure.communication.administration.models.ReleaseResponse;
-import com.azure.communication.administration.models.UpdateNumberCapabilitiesResponse;
-import com.azure.communication.administration.models.NumberConfiguration;
-import com.azure.communication.administration.models.PhoneNumberSearch;
-import com.azure.communication.administration.models.UpdateNumberCapabilitiesRequest;
-import com.azure.communication.administration.models.UpdatePhoneNumberCapabilitiesResponse;
+import com.azure.communication.administration.models.*;
```

Comment:
I think this change (models.*) will give a spot check failure, so you would probably have to go back to listing all of them and adding the newer ones.